spring-cloud-azure-samples/azure-appconfiguration-conversion-sample/complete/src/main/resources/bootstrap.properties

```diff
@@ -0,0 +1,2 @@
+spring.cloud.azure.appconfiguration.stores[0].connection-string=${CONFIG_STORE_CONNECTION_STRING}
+          
```

Comment:
I think there is some extra whitespace on this second line that is tripping up checkstyle.